lib/kcerts/ssh_test.go

```diff
@@ -37,11 +37,20 @@ func TestAddSSHCAToClient(t *testing.T) {
 
 // TODO(adam): test cache failures and edge cases
 func TestStartSSHAgent(t *testing.T) {
+	tmpDir, err := ioutil.TempDir("", "en")
+	assert.Nil(t, err)
+	os.Mkdir(tmpDir+"/.config", 0700)
+	os.Mkdir(tmpDir+"/.config/enkit", 0700)
```

Comment:
both Mkdir should not be necessary, given our code above? which invokes mkdirall?

Reply:
Removed.